tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -207,28 +215,28 @@ impl<T> UnboundedReceiver<T> {
     /// async fn main() {
     ///     let mut buffer: Vec<&str> = Vec::with_capacity(2);
     ///     let limit = 2;
-    ///     let (tx, mut rx) = mpsc::unbounded_channel();
+    ///     let (tx, mut rx) = mpsc::channel(100);
```

Comment:
This is the docs for the unbounded channel?

Reply:
Oh apologies, I've copied a bit too much. I have fixed that.